SlaveWAMPServer.js

```diff
@@ -91,31 +107,64 @@ class SlaveWAMPServer extends WAMPServer {
 		}
 	}
 
+	/**
+	 * @param {string} socketId
+	 * @returns {boolean}
+	 * @returns {undefined}
+	 */
 	onSocketDisconnect(socketId) {
 		return delete this.interProcessRPC[socketId];
 	}
 
-	getCall(request) {
-		return get(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, false);
+	/**
+	 * @param {InterProcessRPCRequestSchema}[request={}] request
+	 * @param {Function} cb
+	 * @returns {undefined}
+	 */
+	saveCall(request = {}, cb) {
+		if (!request.socketId || !request.procedure || !request.signature) {
+			throw new Error('Cannot save a call for wrong InterProcessRPCRequest request');
+		}
+		if (!cb) {
+			throw new Error('Cannot save a call without callback');
+		}
+		return setWith(this.interProcessRPC, `${request.socketId}.${request.procedure}.${request.signature}`, cb, Object);
```

Comment:
Very valid point, thank you. You are right, callbacks will stack and will never be resolved. I will open a separate issue to address it.